releases/net472/KnownIssues/706819 - Winforms applications may have problems rendering UI on latest Operating Systems .md

```diff
@@ -0,0 +1,11 @@
+ # Winforms application may have problems rendering application UI and instead, may render a white screen with a red ‘X’ mark across.
+
+## Symptoms
+Winforms application may have problems rendering application UI and instead, may render a white screen with a red ‘X’ mark across.
```

Comment:
- application --> applications
- application UI --> the application UI
- no comma after instead
- see the previous comment about the use of "across"